client/my-sites/plan-features/index.jsx

```diff
@@ -113,6 +113,10 @@ export class PlanFeatures extends Component {
 							initialSelectedIndex={ initialSelectedIndex }
 						>
 							<table className={ tableClasses }>
+								<caption class="screen-reader-text">
+									{' '}
```

Comment:
What is this space for?

Reply:
Those extra curly braces should not be there. Thanks for catching that!